packages/api/src/services/CodeBuild.ts

```diff
@@ -23,6 +23,7 @@ export module CodeBuildClient {
                 projectName: process.env.CODE_BUILD_PROJECT as string,
                 buildspecOverride: generateBuildSpec(params.buildDir),
                 sourceLocationOverride: `${S3_BUCKET}/${artifactGuid}.zip`,
+                privilegedModeOverride: true,
```

Comment:
Why does CodeBuild need priveleged mode? Just curious.

Reply:
I missed your reason that you gave. I do know if that's true or if something changed recently. That was not a requirement previously.

Reply:
Do you think this is okay to leave here? From my understanding, it's now required if you want to bundle lambdas for cdk using docker in CodeBuild.

Reply:
I think this is fine then.